sbnci/thresholds/MC_Reco_All_ComparisonChiSquareTh.txt

```diff
@@ -0,0 +1,113 @@
+# this threshold file uses # as field separator
+# the format is
+# parameter#warning_threshold#failure_threshold
+# the first # separator needs to be appended to the parameter without any space
+
+
```

Comment:
It is good to have all thresholds in the same file.
Maybe it could help to add a comment for each section to describe to which validation those thresholds belongs to.
Maybe also a README file in each folder briefly describing its content could help future maintenance and extension of the current validations.

Reply:
These are both good ideas, I'll add them in!